### PR TITLE
[WIP] Setup an Azure job

### PR DESCRIPTION
**Pull request by kargakis**
@pschiffe @kwoodson

### CODE REVIEW

File: sjb/config/common/test_cases/origin_release_install_azure.yml [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
+parent: 'common/test_cases/origin_release_with_ecosystem.yml'
+extensions:
+  sync:
+    - "openshift,release=master"
+  parameters:
+    - name: "CLUSTER_PROFILE"
+      description: "The subdirectory of openshift/release cluster/test-deploy to use for configuration of the cluster."
+      default_value: "gcp"
```
Comment (kwoodson) [inline sjb/config/common/test_cases/origin_release_install_azure.yml]: is this shared? If this is the azure job then maybe this should be azure as its default. If this is shared then gcp might be ok

File: sjb/config/common/test_cases/origin_release_install_azure.yml [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        sudo systemctl restart docker
+
+        suffix='origin-${component}'
+        suffix="${suffix}:$( cat /data/src/github.com/openshift/aos-cd-jobs/ORIGIN_TAG )"
+        cd cluster/test-deploy/${CLUSTER_PROFILE}/
+        ../../bin/ansible.sh ansible-playbook \
+          -e "openshift_test_repo=${location_url}" \
+          -e debug_level=5 \
+          -e "oreg_url=${OS_PUSH_BASE_REGISTRY}${OS_PUSH_BASE_REPO}${suffix}" \
+          playbooks/gcp/openshift-cluster/launch.yml \
```
Comment (kwoodson) [inline sjb/config/common/test_cases/origin_release_install_azure.yml]: should this be azure?  This should probably be a call to acs-engine or our cluster create, right?

File: sjb/config/common/test_cases/origin_release_install_azure.yml [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        cp admin.kubeconfig /tmp/cluster-admin.kubeconfig
+  post_actions:
+    - type: "script"
+      title: "deprovision test cluster"
+      repository: "release"
+      timeout: 900
+      script: |-
+        trap 'exit 0' EXIT
+        cd cluster/test-deploy/${CLUSTER_PROFILE}
+        ../../bin/ansible.sh ansible-playbook playbooks/gcp/openshift-cluster/deprovision.yml
```
Comment (kwoodson) [inline sjb/config/common/test_cases/origin_release_install_azure.yml]: We do not have a deprovision yet.  This should probably be:
```
az group delete --resource-group <resource group name>
```

Comment (pschiffe) [reply sjb/config/common/test_cases/origin_release_install_azure.yml]: we do have `deprovision.yml` which deletes resource group, so I think this is fine, but it should be `make down`

**Comment by kargakis** [comment]
@pschiffe let me know if you need any help

/close

**Pull request closed without merging**